specs/jsonschema-core.md

```diff
@@ -1205,19 +1194,19 @@ If an implementation has been configured to resolve that identifier to a schema
 via pre-loading or other means, it can be used automatically; otherwise, the
 behavior described in {{failed-refs}} MUST be used.
 
-#### JSON Pointer fragments and embedded schema resources {#embedded}
+#### JSON Pointer fragment identifiers and embedded schema resources {#embedded}
 
-Since JSON Pointer IRI fragments are constructed based on the structure of the
-schema document, an embedded schema resource and its subschemas can be
-identified by JSON Pointer fragments relative to either its own canonical IRI,
-or relative to any containing resource's IRI.
+Since JSON Pointer fragment identifiers are based on the structure of the schema
+document, an embedded schema resource and its subschemas can be identified using
+JSON Pointer IRI fragments relative to either its own IRI, or relative to any
+containing resource's IRI.
 
 Conceptually, a set of linked schema resources should behave identically whether
 each resource is a separate document connected with [schema
 references](#referenced), or is structured as a single document with one or more
 schema resources embedded as subschemas.
 
-Since IRIs involving JSON Pointer fragments relative to the parent schema
+Since IRIs with JSON Pointer fragments relative to the parent schema
 resource's IRI cease to be valid when the embedded schema is moved to a separate
 document and referenced, applications and schemas SHOULD NOT use such IRIs to
 identify embedded schema resources or locations within them.
```

Comment:
```suggestion
Since IRIs with JSON Pointer fragments are relative to the parent schema
resource's IRI, they cease to be valid when the embedded schema is moved to a separate
document and referenced. Because of this, applications and schemas SHOULD NOT use such IRIs to
identify embedded schema resources or locations within them.
```

It took me some time to parse this as a single sentence.  It makes sense, but I think the above is a bit cleaner.

(I haven't bothered to adjust for line length.)

Reply:
Agreed! I found this sentence difficult as well.